ops/README.md

```diff
@@ -0,0 +1,55 @@
+# CGUs-ops
+
+Recipes to setup infrastructure and deploy CGUs app
+
+> Recettes pour mettre en place l'infrastructure et déployer l'application CGUs
+
+## Requirements
+
+- Install [Ansible](https://docs.ansible.com/ansible/latest/installation_guide/intro_installation.html)
+
+### [For developement only] Additionals dependencies
+
+To test the changes without impacting the production server, a Vagrantfile is provided to test the changes locally in a virtual machine. VirtualBox and Vagrant are therefore required.
+
+- Install [VirtualBox](https://www.vagrantup.com/docs/installation/)
+- Install [Vagrant](https://www.vagrantup.com/docs/installation/)
+
+## Usage
+
+To avoid making changes on the production server by mistake, by default all commands will only affect the vagrant developement VM. Note that the VM need to be started before with `vagrant up`.\
```

Comment:
```suggestion
To avoid making changes on the production server by mistake, by default all commands will only affect the Vagrant development virtual machine (VM). Note that the VM needs to be started before with `vagrant up`.\
```